compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -45,7 +45,7 @@ class CompilationTests {
       compileFilesInDir("tests/pos-custom-args/erased", defaultOptions.and("-language:experimental.erasedDefinitions")),
       compileFilesInDir("tests/pos", defaultOptions.and("-Ysafe-init")),
       // Run tests for experimental lightweight lazy vals
-      compileFilesInDir("tests/pos", defaultOptions.and("-Ysafe-init", "-Ylightweight-lazy-vals", "-Ycheck-constraint-deps"), FileFilter.include(TestSources.posLazyValsAllowlist)),
+      compileFilesInDir("tests/pos", defaultOptions.and("-Ysafe-init", "-Ylegacy-lazy-vals", "-Ycheck-constraint-deps"), FileFilter.include(TestSources.posLazyValsAllowlist)),
```

Comment:
Update the comment above.
Also would we now want to run all the `pos` and `run` tests instead of just the ones from the lists?

Reply:
I think that no, we still want to run these tests, but now for legacy instead of the new lazy vals. New lazy vals will be now tested in standard `pos` and `run` tests. But the comment should be updated ofc.